app/apptesting/concentrated_liquidity.go

```diff
@@ -139,6 +140,9 @@ func (s *KeeperTestHelper) SetupConcentratedLiquidityDenomsAndPoolCreation() {
 	// modify authorized quote denoms to include test denoms.
 	defaultParams := types.DefaultParams()
 	defaultParams.IsPermissionlessPoolCreationEnabled = true
-	defaultParams.AuthorizedQuoteDenoms = append(defaultParams.AuthorizedQuoteDenoms, ETH, USDC, BAR, BAZ, FOO, UOSMO, STAKE, WBTC)
 	s.App.ConcentratedLiquidityKeeper.SetParams(s.Ctx, defaultParams)
+
+	poolManagerParams := s.App.PoolManagerKeeper.GetParams(s.Ctx)
+	poolManagerParams.AuthorizedQuoteDenoms = append(poolmanagertypes.DefaultParams().AuthorizedQuoteDenoms, ETH, USDC, BAR, BAZ, FOO, UOSMO, STAKE, WBTC)
```

Comment:
Note to reviewer: Authorized quote denoms was moved as a parameter from CL to poolmanager. Since taker fees were implemented at the poolmanager level and needed access to this parameter, and CL already imports poolmanager as a keeper, it made sense to move this parameter upwards.

Reply:
I don't get why we append to default params, vs setting a list directly here

Reply:
You are right, it doesn't make sense, it was copied from another part, I can change

Reply:
Wait actually this seems fine